wildfly-bot-config-example.yml

```diff
@@ -15,9 +17,9 @@ wildfly:
       notify: [ another-random-person ]
 
   format:                 # validation check for a correct format of the Pull Request
-    title-check:
-      pattern: "\\[WFLY-\\d+\\]\\s+.*|WFLY-\\d+\\s+.*"    # regex pattern must be matched by the title
-      message: "Wrong content of the title!"              # message to display if check does not succeed
+    title:                                      # Enabled by default. You have to disable explicitly.
+      enabled: false                            # if check enabled. By default, this setting is set true
```

Comment:
align starting letter of comments to either lower or uppercase but don't mix them

Reply:
this is boolean property, I am not sure if it should start with capital letter